pkgs/development/python-modules/cupy/default.nix

```diff
@@ -77,7 +89,6 @@ buildPythonPackage rec {
 
   NVCC = "${lib.getExe cudaPackages.cuda_nvcc}"; # FIXME: splicing/buildPackages
   CUDA_PATH = "${cudatoolkit-joined}";
```

Comment:
This probably selects the `.out`s instead of `.dev`s or `.lib`s

Reply:
you were right. I was able to get it to build by throwing everything at the wall in the `symlinkJoin` above.

Reply:
Note that with the .static outputs the closure size will be _huge_ because symlinkJoin forces references to unused outputs

Reply:
any ideas for how to mitigate that (likely for a future PR)?

Reply:
Ideally we'd get rid of the symlinkJoin and move everything to buildInputs